docs/modules/extended_kalman_filter_localization.rst

```diff
@@ -111,15 +115,19 @@ The robot can get x-y position infomation from GPS.
 
 So GPS Observation model is
 
-.. math:: \textbf{z}_{t} = H\textbf{x}_t
+.. math:: \textbf{z}_{t} = g(\textbf{x}_t) = H \textbf{x}_t
 
 where
 
-:math:`\begin{equation*} B= \begin{bmatrix} 1 & 0 & 0& 0\\ 0 & 1 & 0& 0\\ \end{bmatrix} \end{equation*}`
+:math:`\begin{equation*} H = \begin{bmatrix} 1 & 0 & 0 & 0 \\ 0 & 1 & 0 & 0 \\ \end{bmatrix} \end{equation*}`
+
+The observation function states that
+
+:math: `\begin{equation*} \begin{bmatrix} x' \\ y' \end{bmatrix} = g(\textbf{x}) = \begin{bmatrix} x \\ y \end{bmatrix} \end{equation*}`
```

Comment:
Maybe this space is not needed.
```suggestion
:math:`\begin{equation*} \begin{bmatrix} x' \\ y' \end{bmatrix} = g(\textbf{x}) = \begin{bmatrix} x \\ y \end{bmatrix} \end{equation*}`
```

Reply:
In my environment, this space might cause rendering issue:
![SS 2021-10-03 at 16 49 13](https://user-images.githubusercontent.com/3813847/135745095-a9540d83-9f40-4528-9f5b-6842e751138e.png)